BGL/include/CGAL/boost/graph/properties.h

```diff
@@ -64,6 +73,75 @@ enum face_external_index_t   { face_external_index   } ;
     T t;
   };
 
+
+  template <typename T>
+  struct halfedge_property_t
+  {
```

Comment:
All the new tags should be in the `CGAL` namespace as well as `dynamic_property_map`.